store/tikv/split_region.go

```diff
@@ -65,9 +66,9 @@ func (s *tikvStore) SplitRegion(splitKey kv.Key, scatter bool) (regionID uint64,
 		}
 		splitRegion := res.Resp.(*kvrpcpb.SplitRegionResponse)
 		logutil.BgLogger().Info("split region complete",
-			zap.Binary("at", splitKey),
-			zap.Stringer("new region left", splitRegion.GetLeft()),
-			zap.Stringer("new region right", splitRegion.GetRight()))
```

Comment:
@tiancaiamao  how about let `logutil.ToHex` and `hex.EncodeToString` return a Stringer?

zap's Stringer is powerful, `zap.String` will be eval even if Info level is disabled, but `Stringer` will not do that.

Reply:
Done @lysu @jacksp